lib/myaccount/request.ts

```diff
@@ -26,23 +27,44 @@ export async function sendRequest<
   const {
     accessToken: accessTokenObj
   } = oktaAuth.tokenManager.getTokensSync();
-  
-  const accessToken = options.accessToken || accessTokenObj?.accessToken;
+
+  const atToken = options.accessToken || accessTokenObj;
```

Comment:
nit: `aToken`? or still keep `accessToken`?

Reply:
`atToken` can also be undefined, where `accessToken` can't be. I lost this fight against TS and just decided to use multiple variables

Reply:
Does this typing work?
```ts
let accessToken: string | AccessToken | undefined = options.accessToken || accessTokenObj;
// accessToken could be any of the 3 in the union type
if (typeof accessToken === 'undefined') {
  throw new AuthSdkError();
}
// TS should now know accessToken cannot be undefined
```

Reply:
I think this would work if `accessToken` was a `const`. Since it's a `let` (and therefore can be re-assigned) TS cannot guarantee the value _isn't_ `undefined` when I try to use the variable (when a `undefined` value isn't allowed)